typescript/tsconfig.node20.json

```diff
@@ -0,0 +1,10 @@
+{
+  "$schema": "https://json.schemastore.org/tsconfig",
+  "extends": "./tsconfig.base.json",
+  "compilerOptions": {
+    "lib": ["ES2023"],
+    "module": "node16",
```

Comment:
The inconsistent case here is strange. Might as well stick with all lower case or all camel case

Reply:
good point. i think i default to whatever json schema hints at. I'll make it consistent.